src/xcomponentAPI.ts

```diff
@@ -23,12 +23,12 @@ class XComponentAPI {
         }).bind(this));
     };
 
-    createSession(xcApiFileName: string, serverUrl: string, createSessionListener: (error: Error, session: Session) => void, reconnectionAfterError: boolean = false, reconnectionIntervalSeconds: number = 10): void {
-        this.connection.createSession(xcApiFileName, serverUrl, createSessionListener, reconnectionAfterError, reconnectionIntervalSeconds);
+    createSession(xcApiFileName: string, serverUrl: string, createSessionListener: (error: Error, session: Session) => void, deconnectionErrorListener?: (closeEvent: CloseEvent) => void): void {
```

Comment:
you already have an error listener `createSessionListener: (error: Error, session: Session)`, no need to add another one.